build_scripts/appimage/AppRun

```diff
@@ -13,6 +13,10 @@ export TKPATH="${TK_LIBRARY}"
 # Export SSL certificate
 export SSL_CERT_FILE="${APPDIR}/opt/_internal/certs.pem"
 
+if [ "$1" == "service" ] ; then
+    exec env BASH_SOURCE="${APPIMAGE:-$0} service" "${APPDIR}/install-infection-monkey-service.sh" ${@:2}
```

Comment:
I'm not sure why you need to set `BASH_SOURCE` here. I'm also not sure it's actually being set properly. What about:

```suggestion
    exec bash "${APPDIR}/install-infection-monkey-service.sh" ${@:2}
```

Reply:
It was behaving as expected as I was trying to keep `install-infection-monkey-service.sh` separated from `AppRun` and remain executable by itself.
But then with the `-z ${APPIMAGE}` below that wouldn't make sense. 
I'll take care of it